casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -80,7 +80,7 @@ object InterpreterUtil {
           Log[F].warn(
             s"Computed pre-state hash ${PrettyPrinter.buildString(computedPreStateHash)} does not equal block's pre-state hash ${PrettyPrinter
               .buildString(preStateHash)}"
-          ) *> Right(none[StateHash]).leftCast[BlockException].pure[F]
+          ) >> Right(none[StateHash]).leftCast[BlockException].pure[F]
```

Comment:
I wonder what's the reasoning behind this change since `*>` is only dangerous when there is recursion, right?

Reply:
Also when rhs depends on the (side)effect of lhs (in `lhs *> rhs` there's no guarantee which of the effects is going to be executed first). I'd like to use `>>` by default simply because the gotchas are subtle and can surface without being noticed.